src/main/webapp/app/entities/ttn/ttn-update.tsx

```diff
@@ -208,7 +236,15 @@ export class TTNUpdate extends React.Component<ITTNUpdateProps, ITTNUpdateState>
                     <Label for="id">
                       <Translate contentKey="global.field.id">ID</Translate>
                     </Label>
-                    <AvInput id="ttn-id" type="text" className="form-control" name="id" required readOnly />
+                    <AvInput
+                      id="ttn-id"
+                      type="text"
+                      className="form-control"
+                      name="id"
+                      required
+                      readOnly
+                      value={isNew ? null : this.props.tTNEntity.id}
+                    />
                   </AvGroup>
                 ) : null}
```

Comment:
Please eliminate ternary operation and just use `&&`, I've left the same comment below or abow, have a look at this please, now code seems no so neat.